pages/projects.tsx

```diff
@@ -25,38 +23,45 @@ function Projects({ projects, githubColors }: ProjectsProps): JSX.Element {
           title="projects | open source at ACM at UCLA"
           description="a heads-up overview of the projects that power ACM at UCLA"
           openGraph={{
-            images: [{
-              url: 'https://opensource.uclaacm.com/logo.png',
-              width: 1200,
-              height: 1200,
-              alt: 'The ACM at UCLA logo',
-            }],
+            images: [
+              {
+                url: 'https://opensource.uclaacm.com/logo.png',
+                width: 1200,
+                height: 1200,
+                alt: 'The ACM at UCLA logo',
+              },
+            ],
             site_name: 'open source at ACM at UCLA',
           }}
         />
-        <h1>
-          projects
-        </h1>
+        <h1>check out current UCLA open source projects</h1>
```

Comment:
<img width="485" alt="image" src="https://github.com/uclaacm/opensource/assets/93846021/f663a9dc-bb62-4206-a487-f9c03b876251">

I think we want to change the description on the card link on the landing page to "Check out current UCLA open source projects!", not the title of the projects page.